build/files.js

```diff
@@ -122,7 +117,7 @@ const headRegexp = /(^module.exports = \w+;?)/m
     ]
 
     , isBufferReplacement = [
-      /(\w+) instanceof Buffer/
+      /(\w+) instanceof Buffer/g
```

Comment:
the lack of `g` here caused a bunch of bugs with encoding and is why we probably want to publish 2.0.1 soon.